mbof/migrations/0002_event_title.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9 on 2016-07-06 14:17
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('mbof', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='event',
+            name='title',
+            field=models.CharField(default='Untitled', max_length=50),
```

Comment:
The field will be "Untitled" if the entry in event table already exists OR if the value isn't provided in the API call for creating the entry in the event table: https://docs.djangoproject.com/en/1.9/ref/models/fields/